spec/features/user_updates_site_spec.rb

```diff
@@ -0,0 +1,65 @@
+require "rails_helper"
+
+feature "user updates site", %{
+  As an authenticated user
+  I want to update an item's information
+  So that I can correct errors or provide new information
+  } do
+
+  scenario 'authenticated user sees edit button on show page' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.